File: src/main/java/com/dipol/pdfconvert/data/entity/Da02.java
package com.dipol.pdfconvert.data.entity;

import lombok.Data;

import javax.persistence.*;

import java.util.Date;

@Entity
@Data
public class Da02 {
    @EmbeddedId
    Da02PK da02PK;
    @Basic
    @Column(name = "sort_order")
    private Integer sortOrder;
    @Basic
    @Column(name = "attr")
    private String attr;
    @Basic
    @Column(name = "recode_time")
    private String recodeTime;
    @Basic
    @Column(name = "recode_value")
    private String recodeValue;
    @Basic
    @Column(name = "add_user")
    private String addUser;
    @Basic
    @Column(name = "add_date")
    private Date addDate;
    @Basic
    @Column(name = "add_ip")
    private String addIp;
    @Basic
    @Column(name = "update_user")
    private String updateUser;
    @Basic
    @Column(name = "update_date")
    private Date updateDate;
    @Basic
    @Column(name = "update_ip")
    private String updateIp;
    @Basic
    @Column(name = "bigo")
    private String bigo;

}

File: vue/src/store/da02.ts
import { defineStore } from 'pinia'

import {getDa02List, getDa02ListAll, Page, writeDa02, getDa02EventNo} from "../api";

export interface da02Type {
    da02PK:da02PkType
    sortOrder: Number
    subOrder: Number
    attr: String
    recodeTime: String
    recodeValue: String
    addUser: String
    addDate: Date
    addIp: String
    updateUser: String
    updateDate: Date
    updateIp: String
    bigo: String

}
export interface da02PkType {
    accNo: String
    eventNo: Number
    dssadCode?: String

}
export const Da02Store = defineStore("Da02Store",
    {
        state: () => ({
            da02List:[] as da02Type[],
            eventNo:[] as Number[]
        }),
        getters: {


        },
        actions: {
            getDa02List(params : Page) {
                return getDa02List(params)
            },
            getDa02ListAll(params : da02PkType) {
                return getDa02ListAll(params)
            },
            delete(params : String) {
                return null
            },
            write(params:da02Type) {
                return writeDa02(params)
            },
            getDa02EventNo() {
                const result=getDa02EventNo()
                result.then((response) => {
                    this.eventNo=response
                }).catch((error) => {
                    console.log(error)
                })
            },

        }
    })